### ISSUE TITLE
[Accessibility] Lack of "Select All" Functionality for Surveys in Map/Data View

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4976]
## Describe the bug
 There is lack of a `"select all"` functionality for surveys in the map view/ data view on the web platform. Specifically, when you have cleared all the survey selections and want to select them all again, there is currently no functionality provided to do so efficiently. This is an inconvenience for users who need to quickly re-select all the surveys.

## Where was the bug observed
This issue was observed in the map view of the web platform, accessible at the local host URL: http://localhost:4200/map and http://localhost:4200/feed

## To Reproduce

1. Launch the local host at http://localhost:4200/.
2. Navigate to the mapview/dataview to view the surveys displayed alongside the sidebar.
3. Clear all the survey selections.
4. Attempt to select all the surveys again.

## Expected behavior

- It would be to provide a convenient "select all" functionalityhat allows users to easily re-select all the surveys after they have been cleared. 

## Walk through
[Screencast from 08-07-2024 11:44:50 ASUBUHI.webm](https://github.com/ushahidi/platform/assets/124133577/bbb0804b-9309-4e7e-b671-4dcd4b12caf2)

**Comment by Angamanga** [comment]
Closing but keeping in Linear: https://linear.app/ushahidi/issue/USH-1475/[accessibility]-lack-of-select-all-functionality-for-surveys-in
